src/app/views/sidebar/history/history-utils.ts

```diff
@@ -24,9 +24,12 @@ export const removeHistoryData = async (historyItem: IHistoryItem) => {
   return true;
 };
 
-export const bulkRemoveHistoryData = async (listOfKeys: string[]) => {
+export const bulkRemoveHistoryData = async (listOfKeys: string[], all?: boolean) => {
```

Comment:
I decided to reuse this function to clear all history, but there's  another option, import localForage in src/app/services/actions/auth-action-creators.ts and run the `localForage.clear` function and then  update the state with an empty array ` dispatch({
        type: REMOVE_ALL_HISTORY_ITEMS_SUCCESS,
        response: [],
      }); ` and return an empty array in `src/app/services/reducers/request-history-reducers.ts`

if we don't want to iterate through all the key see below 👇